code/game/machinery/recharger.dm

```diff
@@ -3,19 +3,23 @@
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "recharger0"
 	desc = "A charging dock for energy based weaponry."
-	anchored = 1
+	anchored = TRUE
 	use_power = IDLE_POWER_USE
 	idle_power_usage = 4
 	active_power_usage = 200
 	pass_flags = PASSTABLE
-	var/obj/item/charging = null
-	var/using_power = FALSE
+
 	var/list/allowed_devices = list(/obj/item/gun/energy, /obj/item/melee/baton, /obj/item/modular_computer, /obj/item/rcs, /obj/item/bodyanalyzer)
 	var/icon_state_off = "rechargeroff"
 	var/icon_state_charged = "recharger2"
 	var/icon_state_charging = "recharger1"
 	var/icon_state_idle = "recharger0"
 	var/recharge_coeff = 1
+	var/power_usage_gun = 250
+	var/power_usage_misc = 200
```

Comment:
I think these would be better as defines, because they're static numbers that are never going to change.

Reply:
Yeah, I made them vars with the idea in mind that they could be edited later for a more efficient recharger or something but in the grand scheme of things it's probably very insignificant. I'll make them defines